rules/cre-2025-0033/opentelemetry-collector.yaml

```diff
@@ -0,0 +1,59 @@
+rules:
+- cre:
+    id: CRE-2025-0033
+    severity: 3
+    title: OpenTelemetry Collector refuses scrape due to memory pressure
+    category: observability
+    author: Prequel
+    description: 'The OpenTelemetry Collector may refuse to ingest metrics during a Prometheus scrape if it exceeds its configured memory limits.
+
+      When the `memory_limiter` processor is enabled, the Collector actively drops data to prevent out-of-memory errors, resulting
+
+      in log messages indicating that data was refused due to high memory usage.
+
+      '
+    cause: 'When the Collector''s memory usage surpasses configured soft or hard thresholds, the memory_limiter processor prevents ingestion
+
+      of new telemetry data. This safeguards the process from crashing but leads to temporary loss of incoming metrics.
+
+      '
+    tags:
+    - otel-collector
+    - prometheus
+    - memory
+    - metrics
+    - backpressure
+    - data-loss
+    - known-issue
+    mitigation: '- Tune the memory_limiter configuration using `limit_mib` and `check_interval`.
+
+      - Increase resource limits for the Collector container (e.g., via Kubernetes).
+
+      - Reduce incoming data volume or adjust scrape intervals to lower memory pressure.
+
+      - Monitor Collector memory usage over time and adjust pipeline design accordingly.
+
+      '
+    references:
+    - https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiterprocessor/README.md
+    - https://github.com/open-telemetry/opentelemetry-collector/issues/8217
+    applications:
+    - name: opentelemetry-collector
+      version: '>=0.60.0'
+      containerName: otel-collector
+    impact: loss of metrics due to backpressure from collector
```

Comment:
Capitalize. Consider expanding metrics to the more general "telemetry" since this also applies to traces, logs, etc.